README.md

```diff
@@ -50,14 +50,15 @@ custom_components
 │   ├── sensor.py
 │   └── strings.json
 ```
+Make sure to set the permissions and ownership so that hass has proper access. Restart the hass service so the integration can be installed.
```

Comment:
```suggestion
Make sure to set the permissions and ownership so that Home Assistant has proper access. Restart the Home Assistant service so the integration can be installed.
```

However, this does _not_ even explain:

1. How to set permissions;
2. What permissions to set ("so that Home Assistant has proper access": what user and `rwx` does it need?).

So I don't think it adds any value.

Reply:
I'm of the opinion that this doesn't even belong in the readme. By default the permissions should already be correct, if that is not the case then you have been messing around with the config or you are using an advanced install where we can assume that someone also knows how to solve it.

I also don't know of any custom integration where this was a thing 🤷🏻‍♂️